domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/domene/iay/inntektsmelding/Inntektsmelding.java

```diff
@@ -41,6 +42,22 @@
 @Table(name = "IAY_INNTEKTSMELDING")
 public class Inntektsmelding extends BaseEntitet implements IndexKey {
 
+    public static final Comparator<? super Inntektsmelding> COMP_REKKEFØLGE = (Inntektsmelding a, Inntektsmelding b) -> {
+        if (a == b) {
+            return 0;
+        }
+        if (a.getKanalreferanse() != null && b.getKanalreferanse() != null) {
+            return a.getKanalreferanse().compareTo(b.getKanalreferanse());
+        } else if (a.getKanalreferanse() != null) {
+            return 1;
+        } else if (b.getKanalreferanse() != null) {
+            return -1;
+        } else {
```

Comment:
Bør ikke disse to if (kanalereferanse() != null) ut ? Ref skalFjerneInntektsmelding() Ditto i IM Builder
Dersom kun en har kanalreferanse må vi nesten se på tidspunkt

Reply:
ok